src/app/_components/Authors/AuthorPills/index.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from "@/app/_components/Authors/AuthorPills/styles.module.css";
+import FeaturedImage from "@/app/_components/FeaturedImage";
+import React from "react";
+import { LinkedInIcon, TwitterIcon } from "@/app/_icons/socialIcons";
+import Link from "next/link";
+
+export function AuthorPill({ large = false, author }) {
+  const { name, featuredImage, linkedIn, x } = author;
+  return (
+    <Link href={`${process.env.NEXT_PUBLIC_SERVER_URL}/authors/${author.slug}`}>
+      <div className={styles.authorPill}>
+        <div className={styles.authorImage}>
+          {featuredImage && <FeaturedImage src={featuredImage.url} />}
+        </div>
+        {name}
+      </div>
+
+      {large && (
+        <div className={styles.socials}>
+          {linkedIn && (
+            <a href={linkedIn}>
+              <LinkedInIcon />
+            </a>
+          )}
+          {x && (
+            <a href={x}>
+              <TwitterIcon />
+            </a>
+          )}
+        </div>
+      )}
+    </Link>
+  );
+}
+
+export function TwoAuthorPill({ authors }) {
```

Comment:
It would be easier if there's only 1 AuthorPill. You could continue having the TwoAuthorPill and VariousAuthorsPill, but I'd suggest only exporting one AuthorPill that receives a user array, checks the length, and internally decides if it should use SingleAuthor, TwoAuthor, or Various Author. That way when you use it elsewhere, you dont need to worry about how which version to use

Reply:
Great idea, i'll implement this, thanks

Reply:
done in https://github.com/subvisual/content-sub/pull/58/commits/90ef400d24d1cbf97c7e636fcdc9d003a8907884 ✅